test/unit/io/stream/fast_istreambuf_iterator_test.cpp

```diff
@@ -0,0 +1,55 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
```

Comment:
Remove this file again. You probably had merge conflicts because you edited this file but it did not exist anymore because another PR renamed this file `fast_istreambuf_iterator_test.cpp -> fast_streambuf_iterator_test.cpp`.